src/__mocks__/View.js

```diff
@@ -9,7 +9,9 @@
 
 'use strict';
 
-var createReactNativeComponentClass = require('createReactNativeComponentClass');
+var createReactNativeComponentClass = require(
+  'createReactNativeComponentClass',
+);
```

Comment:
This is a bit unfortunate since often grep for `require('x')` and I never would thought to include the trailing comma. I suppose this changes with ES modules anyway.